flexget/plugins/sites/serienjunkies.py

```diff
@@ -166,7 +166,7 @@ def find_all_titles(self, search_title):
     def check_language(self, languages):
         # Cut additional Subtitles
         try:
-            languages = languages[:languages.index("+")]
+            languages = languages.split('+', 1)[0]
```

Comment:
i self opened some days ago a similar thread with exactly the same problem as in #1409 here
http://discuss.flexget.com/t/url-rewriting-serienjunkies-failed-substring-not-found/2690

the fix in #1409 from you solved my problem!

![image](https://cloud.githubusercontent.com/assets/2155498/18742288/a2f6bbe8-80b3-11e6-9de3-c911610fefff.png)